### ISSUE TITLE
Needless blank line(s) does not get triggered when there is more than one blank line

### ISSUE THREAD

**Issue by vanniktech** [shyiko/ktlint#108]
Test case:

```kotlin
package com.foo.bar

class Test


```

**Comment by shyiko** [comment]
@vanniktech 

```sh
$ printf "package com.foo.bar\n\nclass Test\n\n\n" > T.kt
$ ktlint --version
0.12.1
$ ktlint --relative T.kt 
T.kt:5:1: Needless blank line(s)
```

**Comment by shyiko** [comment]
Everything seems to be in order.

**Comment by vanniktech** [comment]
When I'm running it through the gradle plugin I still see no errors.

This is my config file:

```
[*.{kt,kts}]
indent_size=2
insert_final_newline=true
```

**Comment by shyiko** [comment]
@vanniktech same thing with `.editorconfig` in place. Could you please report this issue to the gradle plugin owner/repo?

**Comment by vanniktech** [comment]
The following does get triggered for me:

```kotlin
}
 


```

While the one below won't be.

```kotlin
}


```

Could this be an off by one error?

**Comment by shyiko** [comment]
@vanniktech I see. Thank you. It was a combination of 0a262d5115d9feef5226cf1e42281adb9b899d11 and 3b9bb1b5fb42881c94c609884978cbc168a00eb0. Fixed.

**Comment by vanniktech** [comment]
Awesome. Thank you :)
